core/src/test/java/org/apache/iceberg/util/TestStructLikeMap.java

```diff
@@ -79,38 +80,37 @@ public void testMultipleRecord() {
     record3.setField("data", null);
 
     Map<StructLike, String> map = StructLikeMap.create(STRUCT_TYPE);
-    Assert.assertEquals(0, map.size());
+    assertThat(map).isEmpty();
 
     map.putAll(ImmutableMap.of(record1, "1-aaa", record2, "2-bbb", record3, "3-null"));
-    Assert.assertEquals(3, map.size());
-    Assert.assertTrue(map.containsKey(record1));
-    Assert.assertTrue(map.containsKey(record2));
-    Assert.assertTrue(map.containsKey(record3));
-    Assert.assertTrue(map.containsValue("1-aaa"));
-    Assert.assertTrue(map.containsValue("2-bbb"));
-    Assert.assertTrue(map.containsValue("3-null"));
-    Assert.assertEquals("1-aaa", map.get(record1));
-    Assert.assertEquals("2-bbb", map.get(record2));
-    Assert.assertEquals("3-null", map.get(record3));
+    assertThat(map)
+        .hasSize(3)
+        .containsKey(record1)
```

Comment:
we don't need to check for `containsKey` and `containsValue` if we already check for `containsEntry`